Clarify Existential Deposit (#2595)
w3f/polkadot-wiki: docs/build/build-protocol-info.md

## Existential Deposit

> Polkadot, and most Substrate-based chains, use an _existential deposit_ (ED) to prevent dust accounts from bloating chain state.

Reviewer: ```suggestion
Polkadot, and most Substrate-based chains, use an *existential deposit* (ED) to prevent a blockchain bloat.
```

Reviewer: @joepetrowski

Reviewer 2: No. What is a blockchain bloat?
